index.html

```diff
@@ -54,54 +78,97 @@ <h3>Expand your career opportunities with Python</h3>
                     </a>
                     <a href="">
                         <div class="course">
                             <img src="images/thumbnail.png" alt="course thumbnail" class="thumb">
                             <h4>Learn Python: The Complete Python Programming Course</h4>
-                            <p style="font-size: 90%; color: rgb(83, 83, 83);">Mahmoud Aboelsoud</p>
-                            <img src="images/rating.png" alt="course rate" class="rate">
-                            <p><b>E<span>&#8364;</span>679.99</b> <del style="color: rgb(103, 101, 101); font-size: 90%;">E<span>&#8364;</span>999.99</del></p>
+                            <p>Mahmoud Aboelsoud</p>
+                            <div class="rating">
+                                <span><b>4.5</b></span>
+                                <span class="fa fa-star"></span>
+                                <span class="fa fa-star"></span>
+                                <span class="fa fa-star"></span>
+                                <span class="fa fa-star"></span>
+                                <span class="fa fa-star-half-stroke"></span>
+                                <span class="rate">(212,883)</span>
+                            </div>
+                            <p><b>E<span>&#8364;</span>679.99</b> <del>E<span>&#8364;</span>999.99</del></p>
                         </div>
                     </a>
                     <a href="">
                         <div class="course">
                             <img src="images/thumbnail.png" alt="course thumbnail" class="thumb">
                             <h4>Learn Python: The Complete Python Programming Course</h4>
-                            <p style="font-size: 90%; color: rgb(83, 83, 83);">Mahmoud Aboelsoud</p>
-                            <img src="images/rating.png" alt="course rate" class="rate">
-                            <p><b>E<span>&#8364;</span>679.99</b> <del style="color: rgb(103, 101, 101); font-size: 90%;">E<span>&#8364;</span>999.99</del></p>
+                            <p>Mahmoud Aboelsoud</p>
+                            <div class="rating">
+                                <span><b>4.5</b></span>
+                                <span class="fa fa-star"></span>
+                                <span class="fa fa-star"></span>
+                                <span class="fa fa-star"></span>
+                                <span class="fa fa-star"></span>
+                                <span class="fa fa-star-half-stroke"></span>
+                                <span class="rate">(212,883)</span>
+                            </div>
+                            <p><b>E<span>&#8364;</span>679.99</b> <del>E<span>&#8364;</span>999.99</del></p>
                         </div>
                     </a>
                 </section>
             </section>  
         </section>
+        <!-- linking script  -->
+        <script src="js/script.js"></script>
     </body>
 </html>
```

Comment:
add a blank line at the end of the file
```suggestion
</html>

```